plugins/woocommerce/src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -216,12 +223,27 @@ function( $key ) use ( $additional_fields_keys ) {
 
 		$schema = [];
 		foreach ( $address_fields as $key => $field ) {
-			$schema[ $key ] = [
+			$field_schema = [
 				'description' => $field['label'],
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'required'    => true,
 			];
+
+			if ( ! empty( $field['type'] ) && 'select' === $field['type'] ) {
```

Comment:
type should never be empty.

Reply:
OK, so no fallbacks, it must be explicitly set?

Reply:
Not really, but we set it to text if it's not set, so our code make sure it's there.